### DISCUSSION TITLE
TP4 - Ejercicio 3 - Duplica

### DISCUSSION THREAD

**Discussion by JeanFranco99** [INGCOM-UNRN-P1/cursada-2023#131] [Preguntas y respuestas] [answered]
`Ejercicio 3 - Duplica`
`Duplicar el valor de una variable usando punteros`

Hay que duplicar el valor de la variable en otra? O hay que realizar la multiplicacion por dos del valor de la variable?

**Comment by martinvilu** [comment, marked as answer]
Multiplicar en dos.
